server/plugins/good-hoodie.js

```diff
@@ -0,0 +1,63 @@
+const Stream = require('stream')
+
+const log = require('npmlog')
+const qs = require('querystring')
+
+class HoodieTransform extends Stream.Transform {
+
+  constructor () {
+    super({ objectMode: true })
+  }
+
+  _transform (data, enc, next) {
+    if (data.event === 'error') {
+      log.error(
+        data.error.name,
+        new Date(data.timestamp).toISOString(),
+        data.error.message,
+        data.error
+      )
+    }
+
+    if (data.event === 'response') {
+      var path = data.path +
+        (Object.keys(data.query).length ? '?' + qs.stringify(data.query) : '')
+      log.http(
+        data.event,
+        new Date(data.timestamp).toISOString() + ' -',
+        data.source.remoteAddress + ' -',
+        data.method.toUpperCase(),
+        path,
+        data.statusCode,
+        data.responseTime + 'ms'
+      )
+    }
+
+    if (data.event === 'request' || data.event === 'log') {
+      var level = findLogLevel(Object.keys(log.levels), data.tags) || 'verbose'
+      log[level](
+        data.event,
+        new Date(data.timestamp).toISOString(),
+        data.tags.length ? data.tags : data.data,
+        data.tags.length ? data.data : ''
+      )
+    }
+
+    log.silly(data.event, data)
+    next(null)
```

Comment:
Right now that's correct.  There's nothing that's chained after `good-hoodie`.  But passing the data through the callback doesn't hurt and if we should decide to chain something after the `good-hoodie` transform in the future, for example:
```js
        hoodieReporter: [
          {
            module: 'good-squeeze',
            name: 'Squeeze',
            args: [
              {
                ops: '*',
                response: '*',
                log: '*',
                error: '*',
                request: '*'
              }
            ]
          },
          {
            module: '../../../server/plugins/good-hoodie'
          },
          {
            module: 'good-squeeze',
            name: 'SafeJson'
          },
          {
            module: 'good-file',
            args: ['./mylog.out']
          }
        ]
```
then nothing would be written to `mylog.out` because the messages are not passed through `good-hoodie`.